spec.bs

```diff
@@ -70,11 +81,129 @@ dictionary AuctionAdInterestGroup {
 };
 </xmp>
 
+<div algorithm="joinAdInterestGroup()">
+
+The <dfn for=Navigator method>joinAdInterestGroup(|group|, |durationSeconds|)</dfn> method steps are:
+
+1. If [=this=]'s [=relevant global object=]'s [=associated Document=] is not [=allowed to use=] the
+  "join-ad-interest-group" [=policy-controlled feature=], then return [=a promise rejected with=] a
+  "{{NotAllowedError}}" {{DOMException}}.
+1. Let |interestGroup| be a new [=interest group=].
+1. Validate the given |group| and set |interestGroup|'s fields accordingly.
+  1. Set |interestGroup|'s [=interest group/expiry=] to now plus |durationSeconds|.
+  1. Let |ownerUrl| be the result of running the [=URL parser=] on
+    |group|["{{AuctionAdInterestGroup/owner}}"]. If |ownerUrl| is an error, or its [=url/scheme=] is
+    not HTTPS, return [=a promise rejected with=] a {{TypeError}}. Otherwise, set |interestGroup|'s

```

Comment:
Schemes are strings, so:

```suggestion
    not "`https`", return [=a promise rejected with=] a {{TypeError}}. Otherwise, set |interestGroup|'s
```